### ISSUE TITLE
Undefined Method: keyword view

### ISSUE THREAD

**Issue by eriktrom** [ember-animation/liquid-fire#465]
Thrown when after installing ember 2.7.0-canary+5903017d(w/ no feature flags set) + master branch of liquid-fire. Uninstalling liquid-fire removes the error. 

(will track down if have time, but leaving note in meantime)

**Comment by mike-north** [comment]
Possibly related to this PR: https://github.com/emberjs/ember.js/pull/13617

liquid-fire#master does not work with ember#beta now. I recommend prioritizing a fix for this, or we run the risk of losing some ember#beta production apps for compatibility reasons.

**Comment by eriktrom** [comment]
fyi - been using this, been working: https://github.com/ember-animation/liquid-fire/compare/master...eriktrom:remove-legacy-view-support

yeah, that pr @mike-north linked to looks like the culprit, packages/ember-htmlbars/lib/keywords/view.js is gone

**Comment by Cryrivers** [comment]
btw, @eriktrom that pr works as long as you don't need `liquid-modal`

**Comment by martndemus** [comment]
Have you tried running a [bisect](https://github.com/emberjs/ember.js/issues/13846#issuecomment-234133694) on `components/ember#canary`, to find the breaking commit?

Using bisect I found that https://github.com/emberjs/ember.js/commit/2a233df8c6aa387898299d79dd4b2965b4266015 broke `ember-one-way-controls` and `ember-form-for`.

**Comment by mike-north** [comment]
@martndemus happy to give it a shot

**Comment by ef4** [comment]
Yes, this is only an issue for liquid-modal, which I intend to remove anyway -- it makes particularly egregious use of private API and it's an inferior experience to just using liquid-fire with https://github.com/ef4/ember-elsewhere to implement modals.

**Comment by kaiwah** [comment]
Might be late to the party, but had this issue when upgrading to Ember 2.8 -- so I removed dist/* and reinstalled liquid-fire and it seemed to do the trick. Looks like this legacyViews has been fixed in the latest commit.
